mantle/Makefile

```diff
@@ -13,7 +13,7 @@ schema-update:
 	$(MAKE) -C ../tools schema
 
 .PHONY: install
-install: build
+install:
```

Comment:
I fear this may cause someone to accidentally test with stale binaries.

Reply:
Hmm I guess so, though rebuilding everything again at install time potentially as a different user is worse IMO if we're talking about developer UX. Whereas most people learn early on they need to `make` before they `make install`.

Reply:
This isn't really a new issue though. What we had [before this PR](https://github.com/coreos/coreos-assembler/pull/2358#pullrequestreview-733280703) was equally prone to this.

Reply:
Agree, there is just a fundamental conflict between expecting `make install` to do a build and not having `sudo make install` build things as root.  

The only sane solution IMO is to teach everyone to do `makesudoinstall` or equivalent for this.